cmd/security-bootstrapper/entrypoint.sh

```diff
@@ -37,6 +37,7 @@ EDGEX_USER_ID=${EDGEX_USER:-$DEFAULT_EDGEX_USER_ID}
 # which then injecting into all other related containers on other services' entrypoint scripts
 # if the executable is not 'security-bootstrapper'; then we consider it not running the bootstrapping process
 # for the user may just want to debug into the container shell itself
+cp -rpd ${SECURITY_INIT_STAGING}/* ${SECURITY_INIT_DIR}/
```

Comment:
this command itself might fail if the target directory doesn't exist. it seems to me that we are relying on the framework (either docker-compose's volume or kub's persistent volume) to ensure that is the case.

Reply:
If there is not a writable volume mounted at the target directory, this container MUST fail.  Desirable side-effect IMO.

Reply:
ok, that sounds to fit the thing but it may be just less intuitive to a developer, maybe add some comments above to help.

Reply:
I have tested it, the "edgex-init" directory was created and everything on staging copied to edgex-init